website/content/docs/release-notes/v0_9_0.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+page_title: v0.9.0
+description: |-
+  Boundary release notes for v0.9.0
+---
+
+# [Boundary v0.9.0](https://www.boundaryproject.io/downloads)
+
+The release notes below contain information about new functionality available in the Boundary v0.9.0 release.
+To see a granular record of when each item was merged into the Boundary project, please refer to the [Changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md).
```

Comment:
It's not clear what is being suggested here by "a granular record of when each item was merged". The changelog has no information about timing for the various features -- is the idea that you'd then click on the PR links and look at the dates/times?

Reply:
pinging @xingluw for comments on the release notes